tests/test_closing.py

```diff
@@ -432,45 +454,82 @@ def get_mempool_when_size_1():
     close_tx_id = mempool_tx_ids[0]
     fee_mempool = round(mempool[close_tx_id]['fee'] * 10**8)
 
-    # Get the proclaimed closing fee from the two nodes' statuses
+    assert opts['expected_close_fee'] == fee_mempool
 
-    status_agreed_regex = re.compile("agreed on a closing fee of ([0-9]+) satoshi")
 
-    # [fee_from_funder_status, fee_from_fundee_status]
-    fees_from_status = [None, None]
+opts = {
+    'funder_feerate_per_kw': 29006,  # closing fee negotiation starts at 21000
+    'fundee_feerate_per_kw': 27625   # closing fee negotiation starts at 20000
+}
```

Comment:
Using a global isn't really nice (especially since we make an extensive use of the `opts` name in the functional tests). Note that it could be made a function which doesn't begin with `test_` (like `closing_fee_options(initiator, expected_fees)`) so that it doesn't get treated as a test by `pytest`.

Reply:
I agree the global was bad. Moved into the generic test function that's called from the other `test_*()` functions.